hphp/hack/hhi/stdlib/builtins_mysqli.hhi

```diff
@@ -0,0 +1,374 @@
+function mysqli_next_result ($link) {}
+function mysqli_num_fields ($result) {}
+function mysqli_num_rows ($result) {}
+function mysqli_options ($link, $option, $value) {}
+function mysqli_ping ($link) {}
+function mysqli_poll (array &$read = null, array &$write = null, &$error = null, $sec, $usec = 0) {}
+function mysqli_prepare ($link, $query) {}
+function mysqli_report ($flags) {}
+function mysqli_query ($link, $query, $resultmode = MYSQLI_STORE_RESULT) {}
+function mysqli_real_connect (
+	$link, 
+	$host = '', 
+	$user = '', 
+	$password = '', 
+	$database = '', 
+	$port = '', 
+	$socket = '', 
+	$flags = null
+) {}
+function mysqli_real_escape_string ($link, $escapestr) {}
+function mysqli_real_query ($link, $query) {}
+function mysqli_reap_async_query ($link) {}
+function mysqli_release_savepoint ($link ,$name) {}
+function mysqli_rollback ($link) {}
+function mysqli_savepoint ($link ,$name) {}
+function mysqli_select_db ($link, $dbname) {}
+function mysqli_set_charset ($link, $charset) {}
+function mysqli_stmt_affected_rows ($stmt) {}
+function mysqli_stmt_attr_get ($stmt, $attr) {}
+function mysqli_stmt_attr_set ($stmt, $attr, $mode) {}
+function mysqli_stmt_field_count ($stmt) {}
+function mysqli_stmt_init () {}
+function mysqli_stmt_prepare ($stmt, $query) {}
+function mysqli_stmt_result_metadata ($stmt) {}
+function mysqli_stmt_send_long_data ($stmt, $param_nr, $data) {}
+function mysqli_stmt_bind_param ($stmt, $types, &$var1) {}
+function mysqli_stmt_bind_result ($stmt, &$var1, &...$_) {}
+function mysqli_stmt_fetch ($stmt) {}
+function mysqli_stmt_free_result ($stmt) {}
+function mysqli_stmt_get_result ($stmt) {}
+function mysqli_stmt_get_warnings ($stmt) {}
+function mysqli_stmt_insert_id ($stmt) {}
+function mysqli_stmt_reset ($stmt) {}
+function mysqli_stmt_param_count ($stmt) {}
+function mysqli_sqlstate ($link) {}
+function mysqli_stat ($link) {}
+function mysqli_stmt_close ($stmt) {}
+function mysqli_stmt_data_seek ($stmt, $offset) {}
+function mysqli_stmt_errno ($stmt) {}
+function mysqli_stmt_error ($stmt) {}
+function mysqli_stmt_more_results ($stmt) {}
+function mysqli_stmt_next_result ($stmt) {}
+function mysqli_stmt_num_rows ($stmt) {}
+function mysqli_stmt_sqlstate ($stmt) {}
+function mysqli_stmt_store_result ($stmt) {}
+function mysqli_store_result ($link) {}
+function mysqli_thread_id ($link) {}
+function mysqli_thread_safe () {}
+function mysqli_use_result ($link) {}
+function mysqli_warning_count ($link) {}
+function mysqli_refresh ($link, $options) {}
+function mysqli_bind_param ($stmt, $types) {}
+function mysqli_bind_result ($stmt, $types, &$var1) {}
+function mysqli_client_encoding ($link) {}
+function mysqli_escape_string ($link, $query) {}
+function mysqli_fetch ($stmt) {}
+function mysqli_param_count ($stmt) {}
+function mysqli_get_metadata ($stmt) {}
+function mysqli_send_long_data ($stmt, $param_nr, $data) {}
+function mysqli_set_opt ($link, $option, $value) {}
+
+define ('MYSQLI_READ_DEFAULT_GROUP', 5);
```

Comment:
I think those should be 

    const int MYSQLI_READ_DEFAULT_GROUP = 5;

etc.